### PR TITLE
Requesting a pull to mozilla:v1 from k88hudson:webview

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
R @Pomax

**Comment by Pomax** [comment]
looks good to me

File: www_src/main.less [hunk 1]
```diff
@@ -35,3 +35,60 @@ body {
...
+  padding: 10px;
+  label, input, button {
+    width: 100%;
+    display: block;
+    margin-bottom: 10px;
+    padding: 10px;
+  }
+}
+
+.tile.map-tile {
```
Comment (Pomax) [inline www_src/main.less]: namespace to `.tile` and merge with above rule?

File: www_src/pages/editor/editor.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  render: function () {
+    return <div>
+      <div className="editor-preview">
+        <img src="../../img/toucan.svg" />
+      </div>
+      <div className="editor-options">
+        <button>Change Image</button>
+        <label>Transparency</label>
+        <input type="range" />
```
Comment (Pomax) [inline www_src/pages/editor/editor.jsx]: `range` without min/max/value?

File: www_src/pages/editor/editor.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <label>Corners</label>
+        <input type="range" />
```
Comment (Pomax) [inline www_src/pages/editor/editor.jsx]: same as above. I think `range` has required properties.

File: www_src/pages/map/map.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+var render = require('../../lib/render.jsx');
+var Link = require('../../components/link/link.jsx');
+
+var Map = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+  render: function () {
+    return <div>
+      <Link className="tile map-tile" url="/project/123" href="/pages/project">
```
Comment (Pomax) [inline www_src/pages/map/map.jsx]: `[,,].map(e => return <Link></Link>)` =D

File: www_src/pages/section-1/section-1.jsx [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
 var render = require('../../lib/render.jsx');
+var Link = require('../../components/link/link.jsx');
 
 var Main = React.createClass({
   render: function () {
     return (
-      <div>
-        <div className="tile"><img src="../../img/demo.png" /></div>
+      <div className="demo">
+        <Link url="/map/123" href="/pages/map" className="tile tile-demo">
```
Comment (Pomax) [inline www_src/pages/section-1/section-1.jsx]: The previous stubs have `<Link>` inside the "inner" div. Was using this different ordering intentional?

Comment (k88hudson) [inline www_src/pages/section-1/section-1.jsx]: yes, in this case the whole thing needs to be clickable, whereas in the other one only the toucan is clickable

File: www_src/static/img/toucan.svg [hunk 6]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="208px" height="147px" viewBox="0 0 208 147" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.2.2 (9983) - http://www.bohemiancoding.com/sketch -->
+    <title>Untitled 3</title>
```
Comment (Pomax) [inline www_src/static/img/toucan.svg]: Surely this is `<title>Tucker</title>` =P

File: www_src/pages/map/map.jsx (hunk 4 above)
Comment (k88hudson) [inline www_src/pages/map/map.jsx]: ah, true!

**Pull request merged**